src/note.c

```diff
@@ -266,9 +263,6 @@ static void loadFont(void)
 
 static char *parseText(char **token, char const *const end)
 {
-    assert(NULL != *token);
-    assert(NULL != end);
```

Comment:
`end` isn't dereferenced below.

Reply:
`**token` is, and then there's a line with `*token != end` below that will be an infinite loop if `end` is `NULL`